sql-cli/config/astro-cli.json

```diff
@@ -4,28 +4,32 @@
       "preRelease": true,
       "sqlCliVersion": ">0.2.2"
     },
-    "1.8": {
+    "1.8.0": {
       "preRelease": false,
       "sqlCliVersion": ">0.2, <0.3"
     },
-    "1.9": {
+    "1.9.0": {
       "preRelease": false,
       "sqlCliVersion": ">0.2, <0.3"
     }
   },
   "baseDockerImage": "quay.io/astronomer/astro-runtime:6.0.4-base",
   "compatibility": {
     "0.3.0rc1": {
-      "astroRuntimeVersion": ">=8.0.0",
+      "astroRuntimeVersion": ">=7.2.0",
+      "astroSDKPythonVersion": ">=1.3.1"
+    },
```

Comment:
We can probably delete the pre-releases when they are no longer necessary (both "0.3.0rc1" and "0.3.1a0"), since the Astro CLI shouldn't be using those anymore, WDYT?